---
bool success
---
int32 grooves_number
string process_state